picoquic/picoquic_internal.h

```diff
@@ -74,6 +74,8 @@ extern "C" {
 #define PICOQUIC_MAX_BANDWIDTH_TIME_INTERVAL_MIN 1000
 #define PICOQUIC_MAX_BANDWIDTH_TIME_INTERVAL_MAX 15000
 
+#define PICOQUIC_MINRTT_MARGIN 128
```

Comment:
wonder if we should call it as PICOQUIC_MINRTT_MARGIN_MICROSECONDS or add a comment says its unit is in microseconds

Reply:
We don't say that for the other variables -- microsec is the default.